ui/tests/acceptance/settings/auth/configure/section-test.js

```diff
@@ -41,7 +41,7 @@ module('Acceptance | settings/auth/configure/section', function(hooks) {
     assert.ok(keys.includes('max_lease_ttl'), 'passes max_lease_ttl on tune');
   });
 
-  for (let type of ['aws', 'azure', 'gcp', 'github', 'kubernetes', 'ldap', 'okta', 'radius']) {
```

Comment:
Are the config routes different for the generated auth methods and these 3 are now generated?

Reply:
that's correct! these 3 are now generated. the config routes aren't necessarily different, but now that ldap, okta, and radius are dynamically generated we don't necessarily know how many tabs they will have.